### PR TITLE
Complete javascript-local-storage.

### PR DESCRIPTION
**Pull request by jayb0c**
Screen Shot 2021-08-16 at 7 12 53 PM

### CODE REVIEW

File: javascript-local-storage/data.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+var todos = [];
+
+window.addEventListener('beforeunload', function (event) {
+  var previousTodosJSON = localStorage.getItem('javascript-local-storage');
```
Comment (BrettAlbright) [inline javascript-local-storage/data.js]: so this functionality needs to be outside of the function, as its job is to retrieve the data from local storage when the app loads

File: javascript-local-storage/data.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  var todosJSON = JSON.stringify(todos);
+  localStorage.setItem('javascript-local-storage', todosJSON);
+
+  if (previousTodosJSON !== null) {
```
Comment (BrettAlbright) [inline javascript-local-storage/data.js]: This should also be outside of the function, as it is checking the data retrieved from local storage to make sure it is not null before assigning it to the `todos` variable.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**